src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11173,7 +11182,7 @@ private ParameterSyntax ParseLambdaParameter(bool isFirst, ref bool hasTypes)
                     refOrOutOrParams = this.EatToken();
                 }
 
-                paramType = this.ParseType(true);
+                paramType = this.ParseType();
```

Comment:
Yes, we do need it to condition the test on whether a feature is required to be enabled for the 'ref' keyword to be accepted. For parameters, we never need the feature to be enabled.

---
In reply to: [79702440](https://github.com/dotnet/roslyn/pull/13814#discussion_r79702440) [](ancestors = 79702440,79675184)